views/personal/address-edit.pug

```diff
@@ -21,10 +20,7 @@ block content
     form.cra-form.cra-form--lg(method='post')
       h2 #{__('Your mailing address')}
 
-      div
-        +textInput('Street number', ['w-1-4','d--ib'])(class='w-3-4', value=streetNumber, name='streetNumber' autofocus=(!errors || errors && firstError.param === 'streetNumber' ? true : false))
-
-        +textInput('Street name', ['w-3-4','d--ib'])(class='w-3-4', value=streetName, name='streetName')
+      +textInput('Street address', ['w-3-4','d--ib'])(class='w-3-4', value=streetAddress, name='line1' autofocus=(!errors || errors && firstError.param === 'line1' ? true : false))
 
       +textInput('Apartment number', 'w-1-2', 'Optional')(class='w-3-4', value=aptNumber name='aptNumber')
```

Comment:
You need to change the `name` of this input as well otherwise submitting a new apartment number doesn't update.